contents/blog/decouple-deployment-from-release.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Why you should decouple deployment from release (and how) 
+date: 2023-09-11
+author: ["ian-vanagas"]
+showTitle: true
+rootpage: /blog
+sidebar: Blog
+hideAnchor: true
+featuredImage: ../images/blog/green-blog-image.jpg
+featuredImageType: full
+category: Engineering
+tags:
+ - Guides
+ - Explainers
+---
+
+Releasing a big feature can be nerve-racking. When deploying it, you don't know if it will work in production, break your app, or cause issues. Without the right processes in place, the safety of your releases is a big question mark.
```

Comment:
Love this intro! So much better
```suggestion
Releasing a big feature can be nerve-racking. When deploying it, you don't know if it will work in production, break your app, or cause other issues. Without the right processes in place, the safety of your releases is a big question mark.
```